doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -27,6 +27,51 @@ especially wherever an advanced tactic language is needed. The previous
 implementation of Ltac, described in the previous chapter, will be referred to
 as Ltac1.
 
+Current limitations include:
+
+- There are a number of tactics that are not yet supported in Ltac2 because
+  the interface OCaml and/or Ltac2 notations haven't been written.  See
+  :ref:`defining_tactics`.
+
+- Missing usability features such as:
+
+  - Printing functions are limited and awkward to use.  Only a few data types are
+    printable.
+  - Deep pattern matching and matching on tuples don't work.
+  - If statements on Ltac2 boolean values
+  - A convenient way to build terms with casts is through the low-level API because the
```

Comment:
```suggestion
  - A convenient way to build terms with casts through the low-level API because the
```
The missing feature is the convenient way to build the terms.